src/components/App.js

```diff
@@ -1,17 +1,40 @@
-/* eslint-disable no-unused-vars */
 import React from 'react';
 import '../App.css';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
 import calculate from '../logic/calculate';
 
-function App() {
-  return (
-    <>
-      <Display result="3" />
-      <ButtonPanel />
-    </>
-  );
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+      history: '',
+    };
+  }
+
+  handleClick(name) {
+    const newState = calculate(this.state, name);
+    this.setState(() => ({ ...newState }));
+  }
```

Comment:
- [ ] OPTIONAL - Consider using an arrow function to setState here. Eg: `handleClick = () => {}`

Reply:
Change handleClick to arrow function